app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_classic_infrastructure_credential.rb

```diff
@@ -0,0 +1,38 @@
+class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::IbmClassicInfrastructureCredential < ManageIQ::Providers::EmbeddedTerraform::AutomationManager::TemplateCredential
+  COMMON_ATTRIBUTES = [].freeze
+
+  EXTRA_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('IBM Cloud Classic Infrastructure User Name'),
+      :helperText => N_('The User Name for IBM Cloud Classic Infrastructure.'),
+      :name       => 'userid',
+      :id         => 'userid',
+      :maxLength  => 100,
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+    {
+      :component  => 'password-field',
+      :label      => N_('IBM Cloud Classic Infrastructure API Key'),
+      :helperText => N_('The API key for IBM Cloud Classic Infrastructure.'),
+      :name       => 'password',
+      :id         => 'password',
+      :type       => 'password',
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+  ].freeze
```

Comment:
I renamed these to userid and password so we don't need to define `params_to_attributes` here.